staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -249,31 +219,29 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		return nil
 	})
 
-	s.GenericAPIServer.PrepareOpenAPIService()
-
-	if s.GenericAPIServer.OpenAPIService != nil {
-		s.rootSpec = s.GenericAPIServer.OpenAPIService.GetSpec()
-		if err := s.updateOpenAPISpec(); err != nil {
+	if openApiConfig != nil {
+		s.openAPIAggregator, err = buildAndRegisterOpenAPIAggregator(
+			s.delegateHandler,
```

Comment:
It's a different issue I think. The solution to #49878 might also solve the issue here though. It's the same thing as with the swagger comment. The swagger route has to query all go-restful containers in all delegates on the chain.